samples/interactive-tutorials/README.md

```diff
@@ -43,44 +43,69 @@ If, for some reason, you have decided to proceed with these code samples without
 ### Select your project and enable the Retail API
 
 Google Cloud organizes resources into projects. This lets you
-collect all the related resources for a single application in one place.
+collect all related resources for a single application in one place.
 
 If you don't have a Google Cloud project yet or you're not the owner of an existing one, you can
 [create a new project](https://console.cloud.google.com/projectcreate).
 
-After the project is created, set your PROJECT_ID to a ```project``` variable.
-1. Run the following command in Terminal:
+After the project is created, set your PROJECT_ID to a ```project``` variable:
+
+1. Run the following command in the Terminal:
+
     ```bash
     gcloud config set project <YOUR_PROJECT_ID>
     ```
 
-1. To check that the Retail API is enabled for your Project, go to the [Admin Console](https://console.cloud.google.com/ai/retail/).
+1. Ensure that the Retail API is enabled for your project in the [API & Services page](https://console.cloud.google.com/apis/api/retail.googleapis.com/).
+
+1.  Log in with your user credentials to run a code sample from the Cloud Shell:
+
+    ```bash
+    gcloud auth login
```

Comment:
Shouldn't this be `google auth application-default login`?

https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login